content/23.vaccines-novel.md

```diff
@@ -360,75 +384,104 @@ In the vaccines developed under this approach, the mRNA coding for a stabilized
-Low efficacy against infection is not surprising considering the vaccines are intended to prevent against severe disease, hospitalization and death rather than infection.
 
+<!--Move to evolution
 Vaccine efficacy is not the only pharmacological intervention affected by VOC.
 Some existing therapeutics, including monoclonal antibody treatments like Bamlanivimab (AbCellera Biologics/ Eli Lilly), were ineffective against the Omicron variant.
 Indeed, only Sotrovimab (Vir Biotechnology/GSK) and Tixagevimab (AstraZeneca) to a much lesser extent could effectively neutralize the omicron variant out of 7 tested monoclonal antibodies [@doi:10.1101/2021.12.14.21267772], which has been verified by others [@doi:10.1101/2021.12.12.472269].
 The antigenic shift of the Omicron variant does raise concerns for future VOC and what effects they may have on future vaccines and therapeutics.
+-->
 
-**Serological Response/Boosters:**
+The WHO continues to monitor the emergence of variants and their impact on vaccine efficacy [@url:https://www.who.int/news-room/feature-stories/detail/the-effects-of-virus-variants-on-covid-19-vaccines].
+Previous research in the computational prediction of the efficacy of vaccines targeting the influenza A virus might complement efforts to monitor these types of viral outbreaks [@doi:10.1093/cid/ciy323].
+To adapt, booster shots are now recommended in many places, and vaccines in the near future will likely account for multiple variants and strains of SARS-CoV-2 [@doi:10.1038/s41591-021-01230-y].
 
-**Variants:**
+### Booster Doses
 
-### Special Populations
+Due to waning immunity and viral evolution, boosters have emerged as an important strategy in retaining the benefits of vaccination over time.
+Homologous booster doses have been investigated for most vaccines.
+For example, over 14,000 adults were administered a booster (second) dose of the Janssen Ad26.COV2.S vaccine [@doi:10.1101/2022.01.28.22270043].
+The booster dose was highly efficacious, with severe COVID-19 and hospitalization prevented almost completely in the vaccinated group.
+A booster dose was also found to improve immune response for Sputnik V vaccinees [@doi:10.1101/2021.12.17.21267976].
+For the AstraZeneca vaccine, a different approach was taken.
+In the interest of distributing first doses as widely as possible, the effects of extending the time between the first and second doses was evaluated and  [@doi:10/hx9c].
 
-<!--Quick summary of other trials in children, are all of them analyzing elderly?-->
+In countries such as Israel, a fourth dose of mRNA vaccines have been introduced in response to the Omicron variant and an initial study in healthcare workers show that the additional immunization is safe and immunogenic with antibody titers restored to peak-third dose titers.
```

Comment:
lower O in omicron? I'm not sure, just saw you changed others in the MS for beta etc.